assets/js/modules/reader-revenue-manager/components/settings/SettingsEdit.stories.js

```diff
@@ -20,76 +20,99 @@
  * Internal dependencies
  */
 import { provideModuleRegistrations } from '../../../../../../tests/js/utils';
+import { Grid, Row, Cell } from '../../../../material-components';
 import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
 import SettingsEdit from './SettingsEdit';
 import { publications } from '../../datastore/__fixtures__';
+import { CORE_MODULES } from '../../../../googlesitekit/modules/datastore/constants';
 import {
 	MODULES_READER_REVENUE_MANAGER,
 	READER_REVENUE_MANAGER_MODULE_SLUG,
 } from '../../datastore/constants';
 
 function Template() {
-	return <SettingsEdit />;
+	return (
+		<div className="googlesitekit-layout">
+			<div className="googlesitekit-settings-module googlesitekit-settings-module--active googlesitekit-settings-module--reader-revenue-manager">
+				<div className="googlesitekit-settings-module__content googlesitekit-settings-module__content--open">
+					<Grid>
+						<Row>
+							<Cell size={ 12 }>
+								<div className="googlesitekit-setup-module">
+									<SettingsEdit />
+								</div>
+							</Cell>
+						</Row>
+					</Grid>
+				</div>
+			</div>
+		</div>
+	);
 }
 
 export const Default = Template.bind( {} );
 Default.storyName = 'Default';
 Default.scenario = {};
 
-export const PublicationSelected = Template.bind( {} );
-PublicationSelected.storyName = 'PublicationSelected';
-PublicationSelected.scenario = {};
-PublicationSelected.args = {
+export const PendingVerification = Template.bind( {} );
+PendingVerification.storyName = 'PendingVerification';
+PendingVerification.scenario = {};
+PendingVerification.args = {
 	setupRegistry: ( registry ) => {
-		const publication = publications[ 0 ];
 		registry
 			.dispatch( MODULES_READER_REVENUE_MANAGER )
-			// eslint-disable-next-line sitekit/acronym-case
-			.setPublicationID( publication.publicationId );
+			.selectPublication( publications[ 1 ] );
 	},
 };
 
-export const PublicationSelectedPendingVerification = Template.bind( {} );
-PublicationSelectedPendingVerification.storyName =
-	'PublicationSelectedWithOnboardingStateNotice';
-PublicationSelectedPendingVerification.scenario = {};
-PublicationSelectedPendingVerification.args = {
+export const ActionRequired = Template.bind( {} );
+ActionRequired.storyName = 'ActionRequired';
+ActionRequired.scenario = {};
+ActionRequired.args = {
 	setupRegistry: ( registry ) => {
-		const publication = publications[ 1 ];
 		registry
 			.dispatch( MODULES_READER_REVENUE_MANAGER )
-			// eslint-disable-next-line sitekit/acronym-case
-			.setPublicationID( publication.publicationId );
+			.selectPublication( publications[ 2 ] );
+	},
+};
+
+export const WithoutModuleAccess = Template.bind( {} );
+WithoutModuleAccess.storyName = 'WithoutModuleAccess';
+WithoutModuleAccess.scenario = {};
+WithoutModuleAccess.args = {
```

Comment:
It would be ideal to display an admin name instead of the fallback `Another admin.` However, it's not critical, so we can leave it as is for now.